### ISSUE TITLE
Custom Scroll Bar

### ISSUE THREAD

**Issue by soham-dixit** [covid19india/covid19india.github.io#2158]
**Is your feature request related to a problem? Please describe.**
- No, it isn't related to any problem. 

**Describe the solution you'd like**
- A custom scroll bar with similar colour scheme from the site

**Additional context**
![Custom Scroll Bar GIF](https://user-images.githubusercontent.com/65973895/84821063-298d3a80-b038-11ea-8990-b6811f13d7be.gif)

**Comment by bee-rickey** [comment]
I don't really see any advantage in this.

**Comment by soham-dixit** [comment]
@bee-rickey Yeah, but just for visual changes.

**Comment by w3dg** [comment]
i might make this hppen but there are some other ui color changes being done might as well merge this issue with that

**Comment by balrajsingh9** [comment]
@debopamgupta @Soham7270 
![Screenshot 2020-07-03 at 7 40 39 PM](https://user-images.githubusercontent.com/37571874/86477213-98bb9c00-bd65-11ea-9fb8-989f46202909.png)
![Screenshot 2020-07-03 at 7 40 51 PM](https://user-images.githubusercontent.com/37571874/86477232-9fe2aa00-bd65-11ea-8e30-8f7e9f322506.png)
![Screenshot 2020-07-03 at 7 41 00 PM](https://user-images.githubusercontent.com/37571874/86477252-a709b800-bd65-11ea-828c-a86a1d4f95a3.png)
![Screenshot 2020-07-03 at 7 41 11 PM](https://user-images.githubusercontent.com/37571874/86477260-aa04a880-bd65-11ea-8ba0-aca86609fa3b.png)

I tried the custom scrollbar thing. It just doesn't look good on light background. Also, I have got it working on Chrome and Safari. Firefox still not working. I have to look into that, but see screenshots for ref.

**Comment by soham-dixit** [comment]
@balrajsingh9 Firefox doesn't support the full customizations with scroll bar.

**Comment by balrajsingh9** [comment]
@Soham7270 It should right [check](https://caniuse.com/#search=scrollbar-color)?

**Comment by soham-dixit** [comment]
@balrajsingh9 Nope, I have also worked on the custom scroll bar, it's totally working on Chrome, Safari and Edge. But not on Firefox. As per my knowledge it isn't supported on Firefox.

**Comment by balrajsingh9** [comment]
@Soham7270 After trying for some time and looking here and there on Github, Reddit, and SO, I was able to achieve custom scrollbars on Firefox. I'm running **_v78_**, but it should work in **_v63+_** 

Also in **_MacOS_** I had to do _Settings > General > Scrolling > Always_
Not sure if this works in Windows. 😕 

<img width="1792" alt="Screenshot 2020-07-03 at 8 22 02 PM" src="https://user-images.githubusercontent.com/37571874/86480017-dd960180-bd6a-11ea-9330-125b75fbd08c.png">

Voila 👀 🎊

**Comment by soham-dixit** [comment]
@balrajsingh9 Awesome! Is it working on all browsers now??

**Comment by balrajsingh9** [comment]
So far tested on Chrome, Safari. Although this works on Firefox, but the customization is limited as opposed to the webkit customization for scrollbars.

Let me know if we should proceed with customizing the scrollbars, but I would limit this to dark backgrounds only. @Soham7270

**Comment by soham-dixit** [comment]
Yeah Firefox' webkit for scroll bar is limited @balrajsingh9 

Edit- Maybe we can do a dark black scrollbar for light theme of the site?

**Comment by w3dg** [comment]
@Soham7270 I agree!

**Comment by amuagarwal** [comment]
Add the scroll bar to state list rather than page so the map will keep constant and visible on scrolling of state list. Currently when we scroll to last state the maps get invisible.
![image](https://user-images.githubusercontent.com/7331483/86504781-8e87b500-bdda-11ea-859b-ca1ca9c717e9.png)
